packages/database/src/migrations/1754523298680-updateMedicationPermissionIds.ts

```diff
@@ -0,0 +1,22 @@
+import { QueryInterface } from 'sequelize';
+
+export async function up(query: QueryInterface): Promise<void> {
+  await query.sequelize.query(
+    `
+      UPDATE permissions
+      SET id = REPLACE(id, 'encountermedication', 'medication')
+      WHERE id LIKE '%encountermedication%';
+    `,
+  );
```

Comment:
There was already a migration to update the permission noun EncounterMedication to Medication. But when importing new permissions, the importer checks if a permission already exists by looking for permission id, ie: `role-administration-encountermedication-write-any`. Because the constructed permission id is now `role-administration-medication-write-any` instead of `role-administration-encountermedication-write-any`, the importer thought this permission does not exist yet as it looks for the id `role-administration-medication-write-any`, try to insert, and failed due to unique constraint